src/rpc/blockchain.cpp

```diff
@@ -140,6 +140,35 @@ static int ComputeNextBlockAndDepth(const CBlockIndex* tip, const CBlockIndex* b
     return blockindex == tip ? 1 : -1;
 }
 
+CBlockIndex* ParseHashOrHeight(const UniValue& param, ChainstateManager& chainman) {
```

Comment:
3f166ecc125fce6ccd995687fa16572090a5d099

nit, can be `static` and move `{` to new line.